src/components/ec-radio-btn-group/ec-radio-btn-group.spec.ts

```diff
@@ -0,0 +1,191 @@
+import type { MountingOptions } from '@vue/test-utils';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { mount } from '@vue/test-utils';
+
+import type { CVueWrapper } from '../../../tests/utils/global';
+import { EcRadioBtnGroup } from '../../main';
+import type { RadioButtonGroupProps } from './types';
+
+function mountRadioBtnGroup(props?: Partial<RadioButtonGroupProps>, mountOpts?: MountingOptions<RadioButtonGroupProps>): CVueWrapper {
+  return mount(
+    EcRadioBtnGroup as any, // eslint-disable-line
+    {
+      props: {
+        options: [
+          { value: 'y', label: 'Yes' },
+          { value: 'n', label: 'No' },
+        ],
+        name: 'testingNameProp',
+        ...props,
+      },
+      ...mountOpts,
+    },
+  ) as CVueWrapper;
+}
+
+describe('EcRadioBtn', () => {
+  it('should render properly', () => {
+    const wrapper = mountRadioBtnGroup();
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  describe('props', () => {
+    it('should render the child component (radio-btn) with all the given props', () => {
+      const wrapper = mountRadioBtnGroup({
+        isDisabled: true,
+        isTextInline: true,
+        isGroupInline: true,
+        modelValue: 'y',
+        label: 'A testing label',
+        errorMessage: 'A testing error message',
+      });
+      const givenProps = wrapper.props();
+      const expectedProps = {
+        options: [{ value: 'y', label: 'Yes' }, { value: 'n', label: 'No' }],
+        name: 'testingNameProp',
+        isDisabled: true,
+        isTextInline: true,
+        isGroupInline: true,
+        modelValue: 'y',
+        label: 'A testing label',
+        errorMessage: 'A testing error message',
+      };
+
+      expect(givenProps).toEqual(expectedProps);
+    });
+
+    describe('name', () => {
+      it('should have the same given "name" attribute among all the radio inputs', () => {
```

Comment:
Probably we do not need that `name` prop.

Reply:
if the name is passed, it can be useful to check it's the same among the inputs